app/controllers/podcasts_controller.rb

```diff
@@ -130,4 +132,17 @@ def podcast_params
       ]
     )
   end
+
+  def compose_message(podcast)
+    title = "<#{podcast_url(podcast.id)}|#{sub_escapes(podcast.title)}>"
+    intro = "A new podcast has been created: #{title}"
+    id = "ID: #{podcast.id}"
+    planner = "<#{podcast_planner_url(podcast.id)}|Schedule episodes>"
+    imports = "<#{podcast_imports_url(podcast.id)}|Start a new import>"
+    [intro, id, planner, imports].join("\n")
+  end
+
+  def sub_escapes(text)
+    text.gsub(/[&<>]/, "&" => "&amp;", "<" => "&lt;", ">" => "&gt;")
```

Comment:
these 3 were the only escape symbols that the Slack docs said needed to be substituted. do i need to prepare or consider any other formatting issues?

Reply:
Nope - I think that'll do it.